modules/ROOT/pages/salesforce/salesforce-mktg-connector-reference.adoc

```diff
@@ -1,661 +1,778 @@
 
-* <<modified-object-trigger>>
-* <<new-object-trigger>>
+== Types
+[[ProxyConfiguration]]
+=== Proxy Configuration
 
-[[modified-object-trigger]]
-=== On Modified Object
-`<sfdc-marketing-cloud:modified-object-trigger>`
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Host a| String | Host where the proxy requests is sent. |  | x
+| Port a| Number | Port where the proxy requests is sent. |  | x
+| Username a| String | The username to authenticate against the proxy. |  |
+| Password a| String | The password to authenticate against the proxy. |  |
+| Non Proxy Hosts a| Array of String | A list of hosts against which the proxy should not be used. |  |
+| Ntlm Domain a| String | The domain to authenticate against the proxy. |  |
+|===
 
+[[Tls]]
+=== Tls
 
-==== Parameters
-[%header,cols="20s,20a,35a,20a,5a"]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
-| Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
-| Since a| String |  +++Specify a date in yyyy-MM-dd HH:mm:ss format (that is, 2017-03-17 16:30:40) to retrieve the selected objects.+++ |  |
-| Object Type a| String |  |  | x
-| Primary Node Only a| Boolean |  +++Whether this source should only be executed on the primary node when running in Cluster.+++ |  |
-| Scheduling Strategy a| scheduling-strategy |  +++Configures the scheduler that triggers the polling.+++ |  | x
-| Redelivery Policy a| <<RedeliveryPolicy>> |  +++Defines a policy for processing the redelivery of the same message.+++ |  |
-| Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  |
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | A comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | A comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> |  |  |
+| Key Store a| <<KeyStore>> |  |  |
+| Revocation Check a| * <<StandardRevocationCheck>>
+* <<CustomOcspResponder>>
+* <<CrlFile>> |  |  |
 |===
 
-==== Output
-[%autowidth.spread]
+[[TrustStore]]
+=== Trust Store
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
-|Type |Object
-| Attributes Type a| Any
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
+| Password a| String | The password used to protect the trust store. |  |
+| Type a| String | The type of store used. |  |
+| Algorithm a| String | The algorithm used by the trust store. |  |
+| Insecure a| Boolean | If true, no certificate validations will be performed, rendering connections vulnerable to attacks. Use at your own risk. |  |
 |===
 
-==== For Configurations
-* <<sfdc-marketing-cloud-config>>
+[[KeyStore]]
+=== Key Store
```

Comment:
```suggestion
=== Keystore
```

Reply:
All instances of key store should be keystore. But this is something we need to fix in the source.